### PR TITLE
complete tasks (functions 12586)

### PR DESCRIPTION
**Pull request by Irina-N23**
Task #12586 has been completed.
Just check whether everything is correct in it!

### CODE REVIEW

File: functions/functions12586.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+const countSumOfTwoNumbers = (firstNumber, secondNumber) => +firstNumber + +secondNumber;
+alert(countSumOfTwoNumbers(105, 11));
+
+
+
+// 2. Create a  function of determining the name (if your name, then hello + name).
+const VALID_NAME = "John";
+let userName = prompt("Just input your name!", "")
+
+let greetUser = (userName === VALID_NAME) ?
```
Comment (SiarheiRylach) [inline functions/functions12586.js]: function :)

File: functions/functions12586.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    () => alert(`Hello, ${userName}!`) :
+    () => alert(`Name "${userName}" is not valid.`);
+
+greetUser();
+
+
+
+// 3. Create of calculating the type of argument and output to the console.
+ function logTypeOfArgument(argument) {
+    console.log(typeof argument);
```
Comment (SiarheiRylach) [inline functions/functions12586.js]: better to use return

Comment (Irina-N23) [reply functions/functions12586.js]: Thank you for your feedback! Fixed.

File: functions/functions12586.js (hunk 1 above)
Comment (Irina-N23) [reply functions/functions12586.js]: Thank you for your feedback! Fixed.

**Pull request merged**